src/System.Security.AccessControl/src/System/Security/AccessControl/ObjectSecurity.cs

```diff
@@ -147,6 +143,11 @@ private void UpdateWithNewSecurityDescriptor( RawSecurityDescriptor newOne, Acce
 
         #region Protected Properties and Methods
 
+        /// <summary>
+        /// Gets the security descriptor for this instance.
+        /// </summary>
+        protected CommonSecurityDescriptor SecurityDescriptor => _securityDescriptor;
```

Comment:
Nit: can _securityDescriptor be made readonly?

Reply:
Sure